nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/check_samplesheet.py

```diff
@@ -82,10 +83,7 @@ def check_samplesheet(file_in,file_out):
             if sample not in sample_run_dict:
                 sample_run_dict[sample] = [sample_info]
             else:
-                if sample_info in sample_run_dict[sample]:
```

Comment:
Will need to leave this bit in @ggabernet because it's there to account for the same sample is being provided multiple times e.g. same sample sequenced on different lanes.

Reply:
Ah I see, that makes sense!